src/Assignments/SimpleProblemPage.tsx

```diff
@@ -94,7 +95,7 @@ export const SimpleProblemPage: React.FC<SimpleProblemPageProps> = () => {
                 const currentVersion = _.maxBy(topic.topicAssessmentInfo.studentTopicAssessmentInfo, 'startTime');
                 if (!_.isNil(currentVersion) && !_.isNil(currentVersion?.numAttempts) && !_.isNil(currentVersion.maxAttempts)) {
                     // if the assessment has expired - do NOT allow submissions...
-                    if (!_.isNil(currentVersion.endTime) && currentVersion.endTime >= new Date()) {
+                    if (!_.isNil(currentVersion.endTime) && moment(currentVersion.endTime).isAfter(moment())) {
```

Comment:
Can change this to:
```
currentVersion.endTime?.toMoment().isAfter(moment());
````

Reply:
I think this didnt work before -- because I hadn't merged in your updates.
Change implemented now